cli/commands/create.js

```diff
@@ -184,7 +184,7 @@ CreateCommand.prototype.run = function run(logger, config, cli, finished) {
 			}
 
 			if (cli.argv.alloy !== undefined) {
-				execSync('alloy new ' + cli.argv['workspace-dir'] + '/' + cli.argv.name);
+				execSync('alloy new \'' + path.join(cli.argv['workspace-dir'], cli.argv.name) + '\'');
```

Comment:
This doesn't appear to work on Windows. Possibly because if single quotes instead of double quotes?

Replace the `execSync` import with:

```
{ spawnSync } = require('child_process'),
```

```suggestion
				spawnSync('alloy', ['new', path.join(cli.argv['workspace-dir'], cli.argv.name)], { stdio: 'inherit' });
```